packages/ember-glimmer/lib/ember-views/target-action-support.js

```diff
@@ -0,0 +1,217 @@
+import { Mixin } from 'ember-metal/mixin';
```

Comment:
This duplication seems odd.  What is actually changed from the mixin in ember-views?